e2e/flows/start-appeal.flow.ts

```diff
@@ -37,13 +37,17 @@ export class StartAppealFlow {
         }
     }
 
-    async completeHomeOfficeReference(clickContinue = false, homeOfficeReferenceNumber = '') {
+    async completeHomeOfficeReference(clickContinue = false, ooc = false, homeOfficeReferenceNumber = '') {
         if (homeOfficeReferenceNumber !== '') {
             await this.ccdFormPage.setFieldValue('Home Office Reference/Case ID', homeOfficeReferenceNumber);
         } else {
             await this.ccdFormPage.setFieldValue('Home Office Reference/Case ID', '01234567');
         }
-        await this.ccdFormPage.setFieldValue('Enter the date the decision letter was sent', '{$TODAY}');
+        if (ooc) {
```

Comment:
If it is ooc, why are we setting date sent to Today?

Reply:
Also, how is feature enabled flag used?

Reply:
ooc date sent set to Today-10

Reply:
This feature is only for ooc